src/Resources/ResourceManager/Utilities/BicepUtility.cs

```diff
@@ -0,0 +1,63 @@
+﻿
+using Microsoft.Azure.Commands.Common.Authentication.Abstractions;
+using Microsoft.Azure.Commands.Common.Exceptions;
+using Microsoft.Azure.Commands.ResourceManager.Cmdlets.Implementation;
+using Microsoft.WindowsAzure.Commands.Utilities.Common;
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.ResourceManager.Cmdlets.Utilities
+{
+    public static class BicepUtility
```

Comment:
Prefer to use internal instead of public if possible

Reply:
This utility will be used in other namespaces, so will stay public access